### PR TITLE
Store borrow limits in CreditOracle

### PR DESCRIPTION
**Pull request by vanruch**

### CODE REVIEW

File: contracts/truefi2/TrueFiCreditOracle.sol [hunk 1]
```diff
@@ -20,6 +20,9 @@ contract TrueFiCreditOracle is ITrueFiCreditOracle, UpgradeableClaimable {
     // @dev Track credit scores for an account
     mapping(address => uint8) score;
 
+    // @dev Track credit scores for an account
+    mapping(address => uint256) borrowLimit;
```
Comment (yuchenlintt) [inline contracts/truefi2/TrueFiCreditOracle.sol:24]: Could you change this to `maxBorrowerLimit` to match the spec? `borrowLimit` by itself could confuse users and give them the impression they could borrow up to that amount.

Also don't forget to update the comment!

File: test/truefi2/TrueFiCreditOracle.test.ts [hunk 2]
```diff
@@ -63,4 +63,27 @@ describe('TrueFiCreditOracle', () => {
...
+
+  describe('set and get borrow limits', () => {
+    const firstBorrowLimit = parseEth(100)
+    const secondBorrowLimit = parseEth(200)
+
+    beforeEach(async () => {
+      await oracle.connect(manager).setBorrowLimit(firstAccount.address, firstBorrowLimit)
+    })
+
+    it('only manager can set scores', async () => {
```
Comment (yuchenlintt) [inline test/truefi2/TrueFiCreditOracle.test.ts:75]: s/score/max borrower limit/g

**Review by yuchenlintt — CHANGES_REQUESTED** [review]
Looks good! Could you update naming throughout the PR to match the spec?

File: contracts/truefi2/TrueFiCreditOracle.sol [hunk 3]
```diff
@@ -20,6 +20,9 @@ contract TrueFiCreditOracle is ITrueFiCreditOracle, UpgradeableClaimable {
     // @dev Track credit scores for an account
     mapping(address => uint8) score;
 
+    // @dev Track credit scores for an account
```
Comment (barrutko) [inline contracts/truefi2/TrueFiCreditOracle.sol:23]:
```suggestion
    // @dev Track max borrowing limit for an account
```

**Review by yuchenlintt — APPROVED** [review]
LGTM, thanks!

**Review by MiksuJak — APPROVED** [review]

**Pull request merged**